src/dbnode/persist/fs/retriever.go

```diff
@@ -962,7 +967,7 @@ func (req *retrieveRequest) Clone(
 }
 
 func (req *retrieveRequest) Start() time.Time {
-	return req.start
+	return req.start.ToTime()
```

Comment:
Maybe change to return `UnixNano`

Reply:
Looks like this func is unused, removing it